.github/workflows/precommit.yaml

```diff
@@ -16,6 +16,6 @@ jobs:
       with:
         python-version: 3.8
     - name: Install pre-commit
-      run: pip install pre-commit
+      run: pip install pre-commit==3.3
```

Comment:
Maybe make this pinning same as in `pyproject.toml`?

Reply:
yep, already done